include/wx/defs.h

```diff
@@ -2699,7 +2693,7 @@ typedef HIShapeRef WXHRGN;
 
 #endif // __WXMAC__
 
-#if defined(__WXMAC__)
+#if defined(__WXMAC__) || defined(__APPLE__)
```

Comment:
I think this should be just
```suggestion
#ifdef __DARWIN__
```

We don't need `||` (when `__WXMAC__` is defined, `__APPLE__` definitely is too) and using `__DARWIN__` is more consistent with the test above (I think it and `__APPLE__` are synonyms nowadays).

Could you please confirm that it works for you with this change?

Reply:
Sure, I will verify this tomorrow once back to the machine. Together with an attempt to build Cocoa stdpaths.

P. S. On macOS `__APPLE__` covers all cases, but I have no idea what iOS uses (and whether it should be considered in any given case).